packages/server/__snapshots__/2_browser_path_spec.coffee.js

```diff
@@ -0,0 +1,71 @@
+exports['e2e launching browsers by path fails with bad browser path 1'] = `
+The supplied browser path could not be resolved to a known browser.
+
+Error:
+
+The version string from "/this/aint/gonna/be/found --version" does not match any known browser. The output was:
```

Comment:
I want to improve the error messaging a bit. I think starting with "The version string" is a weird implementation detail that doesn't feel connected to what the user did at all.

If I pass `/foo/bar/baz` then I'd expect something like: 

```
We could not identify a known browser at the path you provided: "/foo/bar/baz"

The output from the command we ran was:

<output>
```

Reply:
I just realized that this error is actually triple stacked.

There's an outer error from `BROWSER_NOT_FOUND_BY_PATH` that inserts the inner error, which then also inserts the actual failure from the child_process.